packages/sodium-fmt/sodium-fmt.1.0.0/opam

```diff
@@ -0,0 +1,42 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Fmt formatters for Sodium"
+description:
+  "sodium-fmt provides a set of formatters to format values of the various types defined by sodium. It also provides utility formatters which will read from bytes and Sodium.bigbytes values and format them as sodium values."
+maintainer: ["~schube/sodium-fmt@lists.sr.ht"]
+authors: ["Victor Schubert"]
+license: "GPL-3.0-or-later"
+depends: [
+  "alcotest" {with-test}
+  "ocaml" {>= "4.13.1"}
+  "dune" {>= "3.0" & >= "3.0.3"}
+  "sodium" {>= "0.6.0"}
+  "fmt" {>= "0.9.0"}
+  "base64" {>= "3.5.0"}
+  "odoc" {with-doc}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+homepage: "https://git.sr.ht/~schube/sodium-fmt"
+bug-reports: "https://lists.sr.ht/~schube/sodium-fmt"
+doc: "https://schube.srht.site/sodium-fmt/index.html"
+dev-repo: "git+ssh://git@git.sr.ht/~schube/sodium-fmt"
```

Comment:
This repository is not accessible through ssh without write access. https should be used instead
```suggestion
dev-repo: "git+https://git.sr.ht/~schube/sodium-fmt"
```